jdbc-adapter/doc/db2-dialect.md

```diff
@@ -0,0 +1,18 @@
+# DB2 Dialect
```

Comment:
It would be great if you could add the sections here that help the user getting started with this dialect:
* JDBC driver: Which JDBC was it tested with
* Adapter script: How does the SQL statement look like
* Create virtual schema: How does the SQL statement(s) look like

As a reference you can have a look at the EXASOL or Hive dialect documentation: https://github.com/EXASOL/virtual-schemas/blob/master/jdbc-adapter/doc/supported-dialects.md

We added this documentation only recently for a few dialects (EXASOL, Hive and Impala). These kind of information turned out to be very important to get started, so we will add documentation for the other dialects asap.

We can also merge the pull request without this if you prefer that we add the documentation - just let us know.